praw/models/reddit/subreddit.py

```diff
@@ -1393,6 +1408,11 @@ def add(
             ``'light'`` or ``'dark'``.
         :param mod_only: (boolean) Indicate if the flair can only be used by
             moderators.
+        :param allowable_content: If specified, most be one of ``'all'``,
+            ``'emoji'``, or ``'text'`` to restrict content to that type.
+            If set to ``'emoji'`` then the ``'text'`` param must be a
+            valid emoji string, for example ``':snoo:'``.
+        :param max_emojis: (int) Maximum emojis in the flair (default: 10).
```

Comment:
The same comment pertaining to distinguishing between PRAW's default, and Reddit's default.